src/test/java/com/wikia/webdriver/testcases/adstests/TestVUAP.java

```diff
@@ -11,16 +12,41 @@
   private static final Dimension DESKTOP_SIZE = new Dimension(1920, 1080);
 
   @Test(
-          groups = "AdsUapDesktop"
+          dataProviderClass = AdsDataProvider.class,
+          dataProvider = "adsVUAPDesktop",
+          groups = "AdsVuapFlow"
   )
-  public void testVideoPlayFlow() {
-    Page page = new Page("project43", "SyntheticTests/VUAP");
+  public void testVideoPlayFlow(String slotName, Page page) {
     AdsBaseObject ads = new AdsBaseObject(driver, urlBuilder.getUrlForPage(page), DESKTOP_SIZE);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "ads" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1481)